src/view/configuration/configuration.jsx

```diff
@@ -20,7 +20,7 @@ import {
   DialogTrigger,
   Dialog,
   Flex,
-  Heading,
+  Heading as HeadingSlot,
```

Comment:
Note that there is now React-Spectrum's `Heading` component and our own typography `Heading` component. I considered naming our typography `Heading` component something like `TypographyHeading` to distinguish. I'm still open to something like that.